tests/phpunit/test-cache.php

```diff
@@ -904,6 +904,21 @@ public function test_wp_cache_init() {
 		$this->assertEquals( $wp_object_cache->cache, $new_blank_cache_object->cache );
 	}
 
+	public function test_redis_connect_custom_database() {
+		global $redis_server;
+		if ( ! class_exists( 'Redis' ) ) {
+			$this->markTestSkipped( 'PHPRedis extension not available.' );
```

Comment:
Isn't `PHPRedis` required to use this plugin? And the README says `You'll also need to enable Redis and the PHPRedis extension in order to run the test suite against Redis`. Tests should fail if the class does not exist, yes?